<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serverconfigurationtab</class>
 <widget class="QWidget" name="serverconfigurationtab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>serverconfigurationtab</string>
  </property>
  <widget class="QTabWidget" name="serverDefn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="serverSettings">
    <attribute name="title">
     <string>ServerSettings</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>510</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="t1Run">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>510</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QLabel" name="ServerStatus_lbl">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>510</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Server Status:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="shardSettings">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>391</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Shard Settings</string>
     </property>
     <widget class="QLabel" name="shardEnabled_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string extracomment="Used for Multi level servers, ie overworld and caves"/>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="text">
       <string>Shard Enabled</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="shardBool">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="masterIPAddr_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Master IP Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="masterIpAddr">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="masterPort_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Master Port</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="masterPort">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>10888</number>
      </property>
     </widget>
     <widget class="QLabel" name="clusterKey_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cluster Key</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clusterKey">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="miscSettings">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>171</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Misc</string>
     </property>
     <widget class="QLabel" name="maxSnapshots_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Snapshots</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="maxSnapshots">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Console Enabled</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="serverStatus">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>510</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Running :(</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="steamSettings">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>201</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Steam Specific Settings</string>
     </property>
     <widget class="QLabel" name="steamGroupOnly_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Steam Group Only</string>
      </property>
     </widget>
     <widget class="QLabel" name="steamGroupId_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Steam Group Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Steam Group Admins</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="steamGroupId">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="steamGroupOnlyBool">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="steamGroupAdminsBool">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gameplaySettings">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>180</y>
       <width>371</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>GamePlaySettings</string>
     </property>
     <widget class="QLabel" name="maxPlayers_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Players</string>
      </property>
     </widget>
     <widget class="QLabel" name="pvp_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PVP</string>
      </property>
     </widget>
     <widget class="QLabel" name="gameMode_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Game Mode</string>
      </property>
     </widget>
     <widget class="QLabel" name="pauseServer_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause Server when no Players</string>
      </property>
     </widget>
     <widget class="QLabel" name="voteKick_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Vote Kick</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>16</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>survival</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>endless </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>wilderness</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="networkSettings">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>391</width>
       <height>381</height>
      </rect>
     </property>
     <property name="title">
      <string>Network</string>
     </property>
     <widget class="QLabel" name="offlineServer_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Offline Server Only</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="offlineOnlyBool">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>260</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tick Rate</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="tickRate">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>320</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>15</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Whitelist Slots</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>350</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>350</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Manage Whitelisted Players</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="serverIPAddr_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>October Crow</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Server IP Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="serverIpAddr">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="serverName_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>October Crow</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Server Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="serverName">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Server Description</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Accept Connections From Same LAN</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Server Playstyle</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>230</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>cooperative</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>competitive</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>social</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>madness</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Server Port</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>10888</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="worldSettings">
    <attribute name="title">
     <string>GameOptions</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>798</width>
          <height>1600</height>
         </rect>
        </property>
        <widget class="QGroupBox" name="world">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>741</width>
           <height>521</height>
          </rect>
         </property>
         <property name="title">
          <string>World</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>19</y>
            <width>721</width>
            <height>491</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="spacing">
            <number>3</number>
           </property>
          </layout>
         </widget>
        </widget>
        <widget class="QGroupBox" name="resources">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>530</y>
           <width>741</width>
           <height>241</height>
          </rect>
         </property>
         <property name="title">
          <string>Resources</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="food">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>780</y>
           <width>741</width>
           <height>241</height>
          </rect>
         </property>
         <property name="title">
          <string>Food</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="animals">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>1080</y>
           <width>741</width>
           <height>241</height>
          </rect>
         </property>
         <property name="title">
          <string>Animals</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="monsters">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>1340</y>
           <width>741</width>
           <height>241</height>
          </rect>
         </property>
         <property name="title">
          <string>Monsters</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
